### ISSUE TITLE
Unable make changes to html files

### ISSUE THREAD

**Issue by JPJPJPOPOP** [jonorthwash/ud-annotatrix#407]
So whenever I make a change to any of the html files in /server/public/html, and then run `npm run build`, the files are reverted back to the original versions and the changes are removed. Do I need to make changes to some scripts before I can edit html files? @keggsmurph21

**Comment by JPJPJPOPOP** [comment]
So my current highly unefficient workaround is to copy the html file I'm changing (luckily I'm only changing one), then run `npm run build` (which will compile the js parts of my changes), then paste the changes back on the html file. Then I'm using locally served static files to test.

**Comment by JPJPJPOPOP** [comment]
Ah, okay. So instead of changing the html files in /server/public/html, what we actually need to change is the corresponding .ejs files in /server/views.

**Comment by kmurphy4** [comment]
> Ah, okay. So instead of changing the html files in /server/public/html, what we actually need to change is the corresponding .ejs files in /server/views.

Yes!  The HTML files are generated from the [EJS](https://ejs.co/) files!  I'm so sorry for not getting back to you sooner on this.  All of the things that get built when you `npm run build` are specified in `gulpfile.js`.  In this case, the relevant bit is:
```js
gulp.task('html',  function(done) {
  compileEJS();
  done();
});
```
